### PR TITLE
Fullbleed article

### PR DESCRIPTION
**Pull request by Lauraa1003**
Test URLs:
- Before: https://main--helix-sportsmagazine--headwirecom.hlx.page/import-test/play/instruction/putting/article/2018/10/get-your-speed-right-to-sneak-in-more-putts
- After: https://fullbleed-article--helix-sportsmagazine--headwirecom.hlx.page/import-test/play/instruction/putting/article/2018/10/get-your-speed-right-to-sneak-in-more-putts

### CODE REVIEW

**Comment by icaraps** [comment]
Looks like you still have some styling issues https://github.com/headwirecom/helix-sportsmagazine/actions/runs/5600286598/jobs/10242455764?pr=61

File: blocks/full-bleed/full-bleed.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  const embeds = ['youtube', 'twitter', 'brightcove'];
+  block
+    .querySelectorAll(embeds.map((embed) => `a[href*="${embed}"]`).join(','))
+    .forEach((embedLink) => {
+      const parent = embedLink.parentElement;
+      const embed = buildBlock('embed', { elems: [embedLink] });
+      parent.replaceWith(embed);
+    });
+
+  // add horizontal line to article
```
Comment (icaraps) [inline blocks/full-bleed/full-bleed.js:102]: Mhh I think this can be done with css by adding a border-bottom to the div using the same selector wdyt ?

Comment (Lauraa1003) [reply blocks/full-bleed/full-bleed.js:102]: Agree, will do that!

**Comment by icaraps** [comment]
@DriftingSands can you pls squash merge the next PR ? It keeps the commit history cleaner IMO ...

 Screenshot 2023-07-19 at 22 00 13

File: blocks/full-bleed/full-bleed.css [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+  .full-bleed.block .article-body div > p {
+    margin-bottom: 60px;
+  }
+}
+
+.full-bleed.block .attribution {
+  font-style: normal;
+  font-weight: 700;
+  letter-spacing: -0.03em;
+  font-size: 14px;
```
Comment (icaraps) [inline blocks/full-bleed/full-bleed.css:110]: @Lauraa1003 it looks like there should be a bigger font-size for desktop can you please double check ?

File: blocks/full-bleed/full-bleed.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+/**
+ * @param {HTMLDivElement} block
+ */
+export default async function decorate(block) {
+  // Template rendering
+  const template = parseFragment(HTML_TEMPLATE);
+
+  // Identify slots
+  const heading = block.querySelector('h1');
```
Comment (icaraps) [inline blocks/full-bleed/full-bleed.js:78]: @Lauraa1003  You can use the shortcut `assignSlot(block, 'heading', 'h1')` from https://github.com/headwirecom/helix-sportsmagazine/blob/main/scripts/scripts.js#L77

**Review by icaraps — APPROVED** [review]

**Comment by icaraps** [comment]
@Lauraa1003  Feel free to merge

**Pull request merged**